content/en/docs/concepts/architecture/nodes.md

```diff
@@ -28,7 +33,7 @@ The [components](/docs/concepts/overview/components/#node-components) on a node
 
 ## Management
 
-There are two main ways to have Nodes added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
+There are two main ways to have Nodes objects added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
```

Comment:
```suggestion
There are two main ways to have Node objects added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
```

Reply:
Reviewed the changes suggested. Thank you - would be applying those shortly.